packages/pluggableWidgets/datagrid-date-filter-web/src/DatagridDateFilter.tsx

```diff
@@ -110,6 +111,13 @@ export default function DatagridDateFilter(props: DatagridDateFilterContainerPro
                         styles={props.style}
                         tabIndex={props.tabIndex}
                         updateFilters={(value: Date | null, type: DefaultFilterEnum): void => {
+                            if (
+                                (value && props.valueAttribute?.value && !isEqual(props.valueAttribute.value, value)) ||
+                                (value ? value : undefined) !== props.valueAttribute?.value
```

Comment:
I assume we have to convert to `undefined` from `null` because `props.valueAttribute?.value` can only be `undefined`. Can we do `value || undefined` though instead of `value ? value : undefined`?